symengine/cwrapper.cpp

```diff
@@ -394,6 +394,17 @@ int basic_neq(const basic a, const basic b)
     return SymEngine::neq(*(a->m), *(b->m)) ? 1 : 0;
 }
 
+int basic_number_sign(const basic s)
+{
```

Comment:
We should finish up #746, then the (Debug time) assert statement will be inside the `rcp_static_cast` function.